tests/find_package/CMakeLists.txt

```diff
@@ -13,8 +13,8 @@ find_package(VulkanHeaders REQUIRED CONFIG)
 
 find_package(VulkanUtilityLibraries REQUIRED CONFIG)
 
-if (NOT TARGET Vulkan::LayerUtils)
-    message(FATAL_ERROR "Vulkan::LayerUtils target not defined!")
+if (NOT TARGET Vulkan::LayerSettings)
+    message(FATAL_ERROR "Vulkan::LayerSettings target not defined!")
 endif()
 
-target_link_libraries(find_package_example PRIVATE Vulkan::LayerUtils)
+target_link_libraries(find_package_example PRIVATE Vulkan::LayerSettings)
```

Comment:
We should make find_package_example a C project to ensure the interface is C compatible. We just need to change the languages to C, and `client.cpp` to `client.c`

We can keep `add_subdirectory_example` as is to ensure the header file compiles with C++ correctly as well.

` Vulkan is a C99 API designed for explicit control of low-level graphics and compute functionality.`

Reply:
```
cmake_minimum_required(VERSION 3.17)

project(TEST_FIND_PACKAGE LANGUAGES C)

add_library(find_package_example STATIC)

# Test c99 support as well as find_package support
target_compile_features(find_package_example PRIVATE c_std_99)

target_sources(find_package_example PRIVATE
    ${CMAKE_CURRENT_LIST_DIR}/../add_subdirectory/client.c
)

# NOTE: Because VulkanHeaders is a PUBLIC dependency it needs to be found prior to VulkanUtilityLibraries
find_package(VulkanHeaders REQUIRED CONFIG)

find_package(VulkanUtilityLibraries REQUIRED CONFIG)

if (NOT TARGET Vulkan::LayerSettings)
    message(FATAL_ERROR "Vulkan::LayerSettings target not defined!")
endif()

target_link_libraries(find_package_example PRIVATE Vulkan::LayerSettings)
```

Reply:
Great idea!

Reply:
Done